src/main/scala/com/stratio/khermes/clients/shell/KhermesConsoleHelper.scala

```diff
@@ -29,10 +29,10 @@ case class KhermesConsoleHelper(client: KhermesClientActor) {
   lazy val reader = createDefaultReader()
 
   parseLines(
-    firstLoad(AppConstants.KhermesConfigNodePath),
-    firstLoad(AppConstants.KafkaConfigNodePath),
-    firstLoad(AppConstants.TemplateNodePath),
-    firstLoad(AppConstants.AvroConfigNodePath)
+    firstLoad(AppConstants.GeneratorConfigPath),
```

Comment:
What does this method do? Would you mind changing the name of the method to be self-explanatory?